.travis.yml

```diff
@@ -7,13 +7,21 @@ services:
 # tox-travis. However, this avoids adding yet another layer of abstraction, and
 # it offers great flexibility. See:
 # https://stackoverflow.com/questions/37016688/how-to-run-tox-with-travis-ci
+#
+# By default, Traivs provides Ubuntu 14.04 VMs. We specify a newer release so
+# that Python 3.7 can be used in testing.
 language: python
 matrix:
   include:
   - python: 2.7
     env: TOXENV=py27
+    dist: xenial
   - python: 3.6
     env: TOXENV=py36
+    dist: xenial
```

Comment:
I suggest moving this out of the matrix and specifying it at the top of the file, underneath ```sudo: required```